storage/database/leveldb_database.go

```diff
@@ -78,14 +79,12 @@ func GetOpenFilesLimit() int {
 	if err != nil {
 		logger.Crit("Failed to retrieve file descriptor allowance", "err", err)
 	}
-	if limit < 2048 {
-		if err := fdlimit.Raise(2048); err != nil {
-			logger.Crit("Failed to raise file descriptor allowance", "err", err)
+	if limit < minFileDescriptors {
+		if err := fdlimit.Raise(minFileDescriptors); err != nil {
+			logger.Crit("Failed to raise file descriptor allowance to the minimum value",
+				"err", err, "minFileDescriptors", minFileDescriptors)
 		}
 	}
```

Comment:
How about removing these lines and make `checkAndRaiseFDLimit` only one to check and raise the file descriptors?